experiments/MERA/test_binary_mera.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019 The TensorNetwork Authors
```

Comment:
change to binary_mera_test.py for consistency.

Reply:
renamed and added a few more tests.